### PR TITLE
Added the tests I already had

### PR DESCRIPTION
**Pull request by jeroen11dijk**
Added the tests I already had and put them in the current rotation, let me know what you think of the naming. It is up to date with the master at the time of making this

### CODE REVIEW

File: rlbot_configs/single_soccar.cfg [hunk 1]
```diff
@@ -64,7 +64,7 @@ Respawn Time = Disable Goal Reset
 # Only total_num_participants config files will be read!
 # Everything needs a config, even players and default bots.
 # We still set loadouts and names from config!
-participant_config_0 = ../example_bots/simple_bot/simple_bot.cfg
+participant_config_0 = C:\Users\Jeroen\Documents\RLBotPythonExample-master\NVDerevo\Derevo.cfg
```
Comment (DomNomNom) [inline rlbot_configs/single_soccar.cfg]: Revert this file and use `alter_config()` instead. 
e.g. https://github.com/RLBot/RLBotTraining/blob/master/run_exercises.py#L50

What I'd like to get to in the near future is so you can have a NVDerevo playlist in this repo but you have some `runner.py` or config which points to your bot but is not checked into RLBotTraining.

**Review by DomNomNom — CHANGES_REQUESTED** [review]

File: rlbot_configs/single_soccar.cfg (hunk 1 above)
Comment (DomNomNom) [reply rlbot_configs/single_soccar.cfg]: I fixed this in commit 6383521.

File: standard_training/exercises/medium_goalie.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        enemy_car = CarState(physics=Physics(location=Vector3(10000, 10000, 10000)))
+        game_state = GameState(ball=ball_state, cars={0: car_state, 1: enemy_car})
+        return game_state
+
+    def make_grader(self) -> Grader:
+        return GoalieGrader()
+
+
+class LineSave(GraderExercise):
+
```
Comment (DomNomNom) [inline standard_training/exercises/medium_goalie.py:31]: Maybe add some documentation as to how this differs from DefendBallRollingTowardsGoal

File: run_exercises.py [hunk 3]
```diff
@@ -21,12 +26,18 @@ def run_easy_exercises():
...
         'Facing away from opponents goal': FacingAwayFromBallInFrontOfGoal(config_path, 200., car_start_y=5100),
-
+        'Defend by catching up to ball': DefendBallRollingTowardsGoal(config_path),
+        'Rolling Shot': RollingTowardsGoalShot(config_path),
+        'Hook Shot': HookShot(config_path),
+        'Dribbling': Dribbling(config_path),
         'BallInFrontOfGoal2': BallInFrontOfGoal(config_path),
         'BallRollingToGoalie': BallRollingToGoalie(config_path),
         'BallRollingToGoalie2': BallRollingToGoalie(config_path),
+        'Line Save': LineSave(config_path),
```
Comment (DomNomNom) [inline run_exercises.py:36]: This seems to be passing instantly. I'll leave it up to you how to fix it but I don't think we should merge this as is.

**Review by DomNomNom — CHANGES_REQUESTED** [review]

**Comment by jeroen11dijk** [comment]
fixed the things you commented on!

**Review by DomNomNom — CHANGES_REQUESTED** [review]
This commit message seems totally unrelated and this commit doesn't do much

**Comment by jeroen11dijk** [comment]
Which commit are you talking about?

**Comment by DomNomNom** [comment]
I'm talking about commit adb4e36

**Review by DomNomNom — APPROVED** [review]

**Pull request merged**